src/frontend/src/optimizer/plan_node/stream_values.rs

```diff
@@ -37,17 +37,15 @@ impl StreamValues {
     /// `StreamValues` should enforce `Distribution::Single`
     pub fn new(logical: LogicalValues) -> Self {
         let ctx = logical.ctx();
-        let mut watermark_columns = FixedBitSet::with_capacity(logical.schema().len());
-        (0..(logical.schema().len() - 1)).for_each(|i| watermark_columns.set(i, true));
```

Comment:
This is semantically correct for a bounded operator, but I'm not sure whether it's useful. 🤔